lib/ansible/galaxy/role.py

```diff
@@ -364,3 +366,103 @@ def spec(self):
         }
         """
         return dict(scm=self.scm, src=self.src, version=self.version, name=self.name)
+
+    def is_valid(self):
+        """
+        Run sanity check validation to ensure role is compliant with what is expected on Galaxy side
+        """
+
+        results = []
+
+        # ROLE001
+        role001_result = {
+            'name': 'ROLE001',
+            'description': 'Role has a meta/mail.yml file',
+            'optional': False,
+            'pass': os.path.exists('%s/meta/main.yml' % self.path)
```

Comment:
why does it need to be `meta/main.yml` ? shouldn't `y(a)ml` be supported? if not any chance you can point me where it says only `.yml` is supported ?